ethcore/private-tx/src/encryptor.rs

```diff
@@ -145,7 +144,7 @@ impl SecretStoreEncryptor {
 		// response is JSON string (which is, in turn, hex-encoded, encrypted Public)
 		let encrypted_bytes: ethjson::bytes::Bytes = result.trim_matches('\"').parse().map_err(|e| ErrorKind::Encrypt(e))?;
 		if !self.unlock_account(&requester, accounts.clone()) {
```

Comment:
Do we really need to `clone()` the Arc here seems that `&*accounts` would suffice. Anyhow, why do we need unlocking at all? Just attempt to decrypt and let it fail.